shared/templates/rsyslog_logfiles_attributes_modify/oval.template

```diff
@@ -75,10 +75,10 @@
     -->
     <ind:pattern operation="pattern match">^[^(\s|#|\$)]+[\s]+.*[\s]+-?(/+[^:;\s]+);*\.*$</ind:pattern>
     <ind:instance datatype="int" operation="greater than or equal">1</ind:instance>
-    <filter action="exclude">state_groupownership_ignore_include_paths</filter>
+    <filter action="exclude">state_{{{ _RULE_ID }}}_ownership_ignore_include_paths</filter>
```

Comment:
We can remove `ownership` word here and on line 81.
`state_{{{ _RULE_ID }}}_ignore_include_paths`.

Reply:
I believe this filter is not necessary. The paths in `include()` or `$IncludeConfig` won't match the `^[^(\s|#|\$)]+[\s]+.*[\s]+-?(/+[^:;\s]+);*\.*$` regex.

Reply:
Thanks :bow: done in [ba94e4c](https://github.com/ComplianceAsCode/content/pull/9789/commits/ba94e4ceee13613c8fc2527e66d9ea265ddc5690)